frontend/src/lib/components/LemonSelect.stories.tsx

```diff
@@ -17,7 +17,11 @@ export default {
 } as ComponentMeta<typeof LemonSelect>
 
 const Template: ComponentStory<typeof LemonSelect> = (props: LemonSelectProps<LemonSelectOptions>) => {
-    return <LemonSelect {...props} />
+    return (
+        <div className="w-full border p-4">
+            <LemonSelect {...props} />
+        </div>
+    )
```

Comment:
Lol 👍

Reply:
_That_'s where *that* went

![homer-hedge](https://user-images.githubusercontent.com/984817/184146835-92300564-07d4-4a9f-a86f-ed15328e2eb2.gif)